.travis.yml

```diff
@@ -3,7 +3,6 @@ language: node_js
 node_js:
   - "0.10.40"
 install:
-  - "npm install -g eslint"
-  - "npm install -g eslint-plugin-meteor"
+  - "npm install"
 script:
-  - "eslint ./"
+  - "npm test"
```

Comment:
> I think this doesn’t work as intented because even if some tests fails the unix return code will be 0 instead of a greater number

As `npm test` is intended exactly for this, it should work. I just verified locally and it worked as it should (without the silent option).

> `eslint ./`

This should no longer be used because it will use the globally installed version of ESLint.

> I guess it would be great if we could integrate the Eslint linter (including the eslint-meteor extension) with the meteor build system, so one could simple meteor lint

I have thought about that and it is definitely possible. I have taken it into account while developing eslint-plugin-meteor. But `meteor lint` is also very slow, because the app has to be built before getting any results. People can run `$ npm run lint` or `$ npm test` before sending PRs. This will be way faster, because it doesn't need to compile the whole app first. When the linter is integrated into the editor properly, it's so much better because you get errors immediately during development. Having a plugin for `meteor lint` is less convenient because the errors are in the console and not where you develop.

Still, somebody could write a meteor build plugin which uses the linter and the ESLint-plugin-Meteor behind the scenes. It would mainly be great for beginners or big projects like yours. I think it's much work for not-that-much benefit. It may be better to educate people on how to use npm properly instead of building tools which obscure everything. :)